### DISCUSSION TITLE
How to increase aria2 download speed?

### DISCUSSION THREAD

**Discussion by erickyun** [anasty17/mirror-leech-telegram-bot#1394] [Q&A]
Hi,

I have a 1GB DDR4 RAM Ubuntu 20.04.5 LTS (GNU/Linux 5.4.0-135-generic x86_64) VPS.
With a 256GB mounted Block Storage Slab.
With a 1000Mbit download and upload speed.

When I try to `/l` leech something with aria2 I get only 2.90MB/s download speed? when I try to leech same link (that is a video file btw) with `/yl` ytdl I get 99.31MB/s download speed?

Could someone please help me? I would appreciate any help. What settings do I need to change tonincrease download speed?

**Comment by anasty17** [comment]
Because there is a difference btw them. With aria2c u use direct download link which is from site and not extracted. But with yt-dlp it will extract the media link so it will act like u watching it not downloading it. So it's not related to the tool in this case it's related to the extracted link. If u manually tried to extract the link from yt-dlp and u downloaded it using aria2c u will get the same high speed

**Reply by erickyun** [reply]
> With aria2c u use direct download link which is from site and not extracted. But with yt-dlp it will extract the media link

My link is direct link from a **video .mkv file**
ytdl can download direct link if it is a video.
And ytdl downloads faster than aria2 with its native downloader.

> If u manually tried to extract the link from yt-dlp and u downloaded it using aria2c u will get the same high speed

Yes correct but right now aria2 has default settings and aria.sh has it own settings.
So I was wondering what settings I need to increase speed?

**Reply by anasty17** [reply]
There is no setting to increase the speed! Send the link so i can test to understand the issue
